### PR TITLE
WIP: conda-build 3 support

### LINKED ISSUE #65: @conda-forge-admin, please rerender
@conda-forge-admin, please rerender.

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #65.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

Fixes #65

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,29 +10,36 @@ source:
...
+  host:
     - python
-    - cython
-    - setuptools
+    #
+    # Pin Cython to workaround a known issue.
+    #
+    # xref: https://github.com/scikit-learn/scikit-learn/issues/10342
+    #
+    - cython 0.27.*
```
Comment (jakirkham) [inline recipe/meta.yaml]: xref: https://github.com/scikit-learn/scikit-learn/issues/10342

**Comment by jakirkham** [comment]
Still seeing problems during the build on macOS w.r.t replacing linkages in `dylib`s. Have raised the macOS build problem as issue ( https://github.com/conda/conda-build/issues/2943 ).

**Comment by rth** [comment]
Thanks for working on this @jakirkham ! Anything I can do to help? If I understand correctly, the current status is that
 - MacOS builds are blocked by https://github.com/conda-forge/scikit-learn-feedstock/pull/66#issuecomment-394133144
 - Linux test failures - I think we could just patch with the fix for the above mentioned issue.

The fixes for test failures should also be available in the upcoming 0.19.2 release / 0.19.X branch (if they aren't it would be good to know). But I guess it would be cleaner to make sparate PRs conda-build 3 update and version upgrade?

**Comment by jakirkham** [comment]
That summary sounds correct.

The macOS issue looks tricky. Wouldn't advise diving into it unless this is particularly interesting, you have a Mac on hand, and some time.

Fixing up the Linux builds here would be very helpful. If you can advise how best to proceed there, that would be great.

Short term if we can't get `conda-build` 3 support, we can stay on `conda-build` 2 for the patch release.

Also if there is a good commit or RC to test for the release, that would be very helpful for ironing out issues early.

**Comment by jakirkham** [comment]
Just as update, this is down to the macOS failure, which was raised upstream as issue ( https://github.com/conda/conda-build/issues/2943 ).

**Comment by jakirkham** [comment]
Closing in favor of PR ( https://github.com/conda-forge/scikit-learn-feedstock/pull/69 ). Can pull other things from here later as needed.

**Pull request closed without merging**